gkroam.el

```diff
@@ -711,7 +711,7 @@ The overlays has a PROP and VALUE."
       (let* ((btn-label (button-label btn))
              (btn-start (button-start btn))
              (btn-end (button-end btn))
-             (new-link (completing-read "Edit link: " nil nil nil btn-label)))
+             (new-link (completing-read "Edit link: " nil nil nil nil nil btn-label)))
```

Comment:
```suggestion
             (new-link (completing-read "Edit link: " nil nil nil btn-label nil btn-label)))
```
```gkroam-edit-link``` is used to edit gkroam link in minibuffer, so it's better to preserve the init-input.